suse/rdma-core.spec

```diff
@@ -513,9 +518,8 @@ rm -rf %{buildroot}/%{_sbindir}/srp_daemon.sh
 %{_udevrulesdir}/90-rdma-hw-modules.rules
 %{_udevrulesdir}/90-rdma-ulp-modules.rules
 %{_udevrulesdir}/90-rdma-umad.rules
-%{_udevrulesdir}/98-rdma.rules
+%{_udevrulesdir}/98-rdma-sriov.rules
 %config %{sysmodprobedir}/50-libmlx4.conf
-%{_libexecdir}/rdma-init-kernel
 %{_libexecdir}/rdma-set-sriov-vf
```

Comment:
redhat/rdma.mlx4.sys.modprobe is not a config file, at least not when installed in /usr/

Reply:
Agreed. Early mistakes in the first packaging. the .conf suffix is misleading